2022-02-23 18:02:05,590 INFO    MainThread:5812 [wandb_setup.py:_flush():71] setting env: {}
2022-02-23 18:02:05,590 INFO    MainThread:5812 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-23 18:02:05,590 INFO    MainThread:5812 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220223_180205-2ccakf5a/logs/debug.log
2022-02-23 18:02:05,591 INFO    MainThread:5812 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220223_180205-2ccakf5a/logs/debug-internal.log
2022-02-23 18:02:05,591 INFO    MainThread:5812 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f3e646bc6d0>
2022-02-23 18:02:05,591 INFO    MainThread:5812 [wandb_init.py:init():404] calling init triggers
2022-02-23 18:02:05,591 INFO    MainThread:5812 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-23 18:02:05,591 INFO    MainThread:5812 [wandb_init.py:init():460] starting backend
2022-02-23 18:02:05,591 INFO    MainThread:5812 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-23 18:02:05,648 INFO    MainThread:5812 [backend.py:ensure_launched():216] starting backend process...
2022-02-23 18:02:05,681 INFO    MainThread:5812 [backend.py:ensure_launched():222] started backend process with pid: 5857
2022-02-23 18:02:05,683 INFO    MainThread:5812 [wandb_init.py:init():469] backend started and connected
2022-02-23 18:02:05,721 INFO    MainThread:5812 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-23 18:02:05,723 INFO    MainThread:5812 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-23 18:02:05,723 INFO    MainThread:5812 [wandb_init.py:init():533] updated telemetry
2022-02-23 18:02:05,818 INFO    MainThread:5812 [wandb_init.py:init():563] communicating current version
2022-02-23 18:02:07,872 INFO    MainThread:5812 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-23 18:02:07,874 INFO    MainThread:5812 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-23 18:02:08,144 INFO    MainThread:5812 [wandb_init.py:init():606] starting run threads in backend
2022-02-23 18:02:13,159 INFO    MainThread:5812 [wandb_run.py:_console_start():1810] atexit reg
2022-02-23 18:02:13,159 INFO    MainThread:5812 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-23 18:02:13,160 INFO    MainThread:5812 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-23 18:02:13,161 INFO    MainThread:5812 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-23 18:02:13,161 INFO    MainThread:5812 [wandb_init.py:init():633] run started, returning control to user process
2022-02-23 18:02:33,460 INFO    MainThread:5812 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 18:02:33,460 INFO    MainThread:5812 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 18:04:25,164 INFO    MainThread:5812 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-23 18:04:25,165 INFO    MainThread:5812 [wandb_run.py:_restore():1752] restore
